packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -132,12 +132,13 @@ export const SelectPopover = styled( Ariakit.SelectPopover )`
 	min-width: min-content;
 
 	/* Animation */
-	animation-duration: ${ ANIMATION_PARAMS.DURATION };
-	animation-timing-function: ${ ANIMATION_PARAMS.EASING };
-	animation-name: ${ slideDownAndFade };
-	will-change: transform, opacity;
-	@media ( prefers-reduced-motion ) {
-		animation-duration: 0s;
+	&[data-open] {
```

Comment:
Is this a piece we can address in a separate PR? 

Looks like `data-open` can be used from `v0.4.0` (see [changelog](https://ariakit.org/changelog#improved-animation-support)).

Reply:
No, this is necessary. Tests fail without this change, and animations are visibly broken without it. This is purely an implementation detail, the behavior remains the same as long as this change is made.